content/desktop/release-notes.md

```diff
@@ -75,6 +75,12 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 
 - Fixed an issue where some `wincred` values were persisted after uninstall. Reported by Javier Yong [@Javiery3889](https://github.com/Javiery3889).
 
+### Known Issues
+
+#### For Windows
+
+- Docker Desktop fails to start with WSL 2.3.11.0, which is included in Windows 11 Insider. To fix this ensure WSL `v2.2.4.0`ins installed. For more information, see [microsoft/WSL#11794](https://github.com/microsoft/WSL/issues/11794)
```

Comment:
nit: s/`v2.2.4.0`ins/`v2.2.4.0` is/

Reply:
typo: "ins installed" -> "installed"

Reply:
Oh no sorry it should be "is installed" :)